src/Evolution/Systems/Cce/CMakeLists.txt

```diff
@@ -54,6 +54,7 @@ spectre_target_headers(
 target_link_libraries(
   ${LIBRARY}
   PRIVATE
+  CceAnalyticSolutions
```

Comment:
This is also in public a few lines below now (I had to add it when to get shared libs working in CI). I don't remember how carefully I check whether or not it needed to be public or private.